modules/ebi-metagenomics/diamond/makedb/environment.yml

```diff
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - bioconda::diamond=2.1.8
```

Comment:
The diamond version for the diamond blastp is [2.1.11](https://github.com/EBI-Metagenomics/nf-modules/blob/321c942ce28fdafef7ffe350697446fc1f218511/modules/ebi-metagenomics/diamond/blastp/environment.yml), so should probably update this one:

Reply:
yeah, we can update that later